bugbot/rules/assignee_no_login.py

```diff
@@ -168,6 +173,60 @@ def get_bz_params(self, date):
 
         return params
 
+    def send_consolidated_email(self, date):
```

Comment:
I think we could use the `Nag` class for this. It will take care of sending the emails and grouping bugs from the same managers together.

See: https://github.com/mozilla/bugbot/blob/7c6a9b302a0ed1b5c5c9cec8403c3ce069620c6e/bugbot/rules/tracked_needinfo.py#L66

Reply:
Still a WIP, but started to the `Nag` class from d6faf7c3c6356c5b8fa7e33e65ef6c5f2dc70469 onward.